### ISSUE TITLE
JS error with wire:model in select

### ISSUE THREAD

**Issue by azurinspire** [livewire/livewire#316]
**Describe the bug**
I'm getting following error when using wire:model with select:
`TypeError: Element.setAttribute requires at least 2 arguments, but only 1 were passed`

Error comes from `specialElHandlers.js` file in where the second attribute of `setAttribute` was dropped in this commit: https://github.com/livewire/livewire/commit/38caadbfdb128252a1251354a4c0e15d0935bf55#diff-3e6ee14443ed0559b0e9e7fb4dfc26b2L7 

I think it shouldn't be dropped:
https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute

**Expected behavior**
No errors

**Desktop (please complete the following information):**
 - OSX  Firefox 69

**Comment by azurinspire** [comment]
Actually the error comes if translation is used as option text and for some reason locales are not in sync on client and server. 

![image](https://user-images.githubusercontent.com/40248184/66219720-f5e9ca80-e6cb-11e9-8129-8695a0254450.png)

![image](https://user-images.githubusercontent.com/40248184/66219735-fbdfab80-e6cb-11e9-86e6-b99461115c74.png)

![image](https://user-images.githubusercontent.com/40248184/66219749-026e2300-e6cc-11e9-897f-99769edcdf5a.png)

**Comment by azurinspire** [comment]
But actually this error happens always if option labels are dynamic for some reason. Don't know if there are really use cases when that would be needed...

Minimal example to reproduce:

```html
<div>
   <select wire:model="test">
        <option value="0">{{ $dynamic }}</option>
        <option value="1">option 1</option>
        <option value="2">option 2</option>
        <option value="3">option 3</option>
   </select>
</div>
```

```php
<?php

namespace App\Http\Livewire;

use Livewire\Component;

class Test extends Component
{
    public $test = 0;
    public $dynamic = 0;

    public function render()
    {
        $this->dynamic++;
        return view('livewire.test');
    }
}
```

**Comment by azurinspire** [comment]
This could be quite normal use case which is causing same error:
```html
<div>
    <select wire:model="test1">
        <option value="0">option 0</option>
        <option value="1">option 1</option>
        <option value="2">option 2</option>
        <option value="3">option 3</option>
    </select>

    <select wire:model="test2">
        @foreach ($options as $option)
            <option value="{{ $option['value'] }}">{{ $option['label'] }}</option>
        @endforeach
    </select>
</div>
```

```php
<?php

namespace App\Http\Livewire;

use Livewire\Component;

class Test extends Component
{
    public $test1 = 0;
    public $test2 = 0;
    public $options = [
        ['value' => 0, 'label' => 'select other first'],
    ];

    public function updatedTest1()
    {
        $this->options = [
            ['value' => 1, 'label' => 'other 1'],
            ['value' => 2, 'label' => 'other 2'],
            ['value' => 3, 'label' => 'other 3'],
        ];
    }

    public function render()
    {
        return view('livewire.test');
    }
}
```

**Comment by calebporzio** [comment]
Sorry, it's hard to follow this issue quickly. Can you provide a simple bit of code I can copy and paste to recreate the issue?

Also, we can definitely add that second param back into specialHandlers.js if that will fix it, I just need clarification that that is the fix, OR I need to reproduce the issue locally to verify myself.

Thanks.

**Comment by azurinspire** [comment]
You should be able to recreate it with the code in my last comment, at least it is giving the error for me. And I'm not sure my first comment was valid, adding the second parameter is not probably the correct fix.

**Comment by allmanaj** [comment]
I've just tested this and adding the second parameter back into the `fromEl.setAttribute` in specialElHandlers.js seems to resolve the issue. I'll open a PR shortly 😁

**Comment by allmanaj** [comment]
opened a PR #326

**Comment by calebporzio** [comment]
Is fixed in latest release: 0.3.17

**Comment by intellow** [comment]
@calebporzio This wasn't actually fixed in the latest release. The update was made in the source file (line 5 of /js/dom/morphdom/specialElHandlers.js), but the update never made it to the compiled js file (line 4617 of /dist/livewire.js) and I'm still seeing the issue on the latest release.

**Comment by calebporzio** [comment]
Fixed now @intellow?

**Comment by intellow** [comment]
Yep! Thanks!
